include/bluetooth/scan.rst

```diff
@@ -80,6 +80,35 @@ Filter modes
 |             | Otherwise, the not found callback is called.                                    |
 +-------------+---------------------------------------------------------------------------------+
 
+Directed advertising
+====================
+
```

Comment:
I think we should document the requirement for privacy here.
By default the zephyr host always protects the scanners privacy, so to support receiving directed advertisement it is recommended to enable the privacy feature. If privacy feature cannot be enabled then the scan with identity option must be enabled to receive directed advertisement, this makes the device trackable while scanning.

Reply:
Right. I added this to doc with references to configuration